src/styles/screen-size-mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin phone {
```

Comment:
I don't really like the idea of creating different mixins files. It would be easier to have one `styles/mixins.scss`